esm/interface/xml-attr.js

```diff
@@ -0,0 +1,19 @@
+import {VALUE} from '../shared/symbols.js';
+import {AMPERSAND,QUOTE,LT,GT} from '../shared/constants.js';
+import {emptyAttributes} from '../shared/attributes.js';
+import {Attr} from './attr.js';
+
+/**
+ * @implements globalThis.Attr
+ */
+export class XmlAttr extends Attr {
+  constructor(ownerDocument, name, value = '') {
+    super(ownerDocument, name, value);
+  }
+
+  toString() {
+    const {name, [VALUE]: value} = this;
+    return emptyAttributes.has(name) && !value ?
+            name : `${name}="${value.replace(AMPERSAND, '&amp;').replace(QUOTE, '&quot;').replace(LT, '&lt;').replace(GT, '&gt;')}"`;
```

Comment:
this won't ever happen in any of my projects, sorry ... here's why: https://github.com/WebReflection/html-escaper#why